docs/COMPILATION.MD

```diff
@@ -186,7 +186,45 @@ Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+<h1>Compilation guide</h1>
+<h4>*First of all you have to have your visual studio installed on your computer.</h4>
+
+<p>When you already have visual studio you have to download ccextractor code <a href="https://github.com/CCExtractor/ccextractor/">CCExtractor</a></p>
+  
+<p>After downloading open your CCExtractor folder and then open windows folder.</p>
+<br>
```

Comment:
not required, someone reading this file, is expected to have ccextractor already, and downloading ccextractor is not window specific.